doc/whats-new.rst

```diff
@@ -48,10 +48,14 @@ New Features
 - :py:meth:`Dataset.swap_dims` and :py:meth:`DataArray.swap_dims`
   now allow swapping to dimension names that don't exist yet. (:pull:`3636`)
   By `Justus Magin <https://github.com/keewis>`_.
-- Extend :py:class:`core.accessor_dt.DatetimeAccessor` properties 
-  and support `.dt` accessor for timedelta 
+- Extend :py:class:`core.accessor_dt.DatetimeAccessor` properties
+  and support `.dt` accessor for timedelta
   via :py:class:`core.accessor_dt.TimedeltaAccessor` (:pull:`3612`)
   By `Anderson Banihirwe <https://github.com/andersy005>`_.
+- Define 1970-01-01 as the default offset for the interpolation index for both
+  normal and CF-time indexes, use microseconds in the conversion from timedelta
```

Comment:
It could be nice also to advertise in this what's new that you've added support for CFTimeIndexes in `interpolate_na`; previously one could only use a DatetimeIndex for times there!

Reply:
Done.